recipe/meta.yaml

```
@@ -65,7 +69,10 @@ outputs:
        - blas_{{ blas_impl }}  # [blas_impl != blas_default_impl]
     requirements:
       build:
-        - {{ compiler('fortran') }}   # [blas_impl == 'accelerate']
+      {% if blas_impl.endswith('accelerate') %}
+        - {{ compiler('c') }}
+        - {{ compiler('fortran') }}
+      {% endif %}
```

Still feel this would have been easier to read and more explicit with the following, but just a nit
```suggestion
        - {{ compiler('fortran') }}   # [blas_impl in ('accelerate', 'new_accelerate')]
        # need to compile wrapper code, see build.sh
        - {{ compiler('c') }}         # [blas_impl == 'newaccelerate']
```